TaskList/app/controllers/tasks_controller.rb

```diff
@@ -9,4 +9,33 @@ def show
     id = params[:id]
     @tasks = Task.where(id: id)
   end
+
+  def new
+    @title = "Add a Task"
+    @task = Task.new()
+  end
+
+  def create
+    Task.create(task_params[:task])
+    redirect_to('/')
+  end
+
+  def destroy
+    Task.delete(params[:id])
+    redirect_to('/')
+  end
+
+  def edit
+    @title = "Edit This Task"
+    id = params[:id]
+    @tasks = Task.where(id: id)
```

Comment:
Per my similar comment on your wave 1 PR, it's definitely preferred to be using the `find` method over `where` because it will prevent unexpected behavior